.eslintrc.js

```diff
@@ -88,7 +89,8 @@ module.exports = {
 
         "no-restricted-imports": ["error", {
             patterns: ["*Events"],
-        }]
+        }],
+        "import/extensions": ["error", "ignorePackages"]
```

Comment:
we need to figure out how packages work for a browser ESM build - but that's a separate issue to this one. I'll have a PR/issue up describing our potential approach later :)